public/data/RohithJN.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "Rohith JN",
+    "type": "personal",
+    "bio": "Junior developer // React.js, Python, APIs",
+    "avatar": "https://github.com/Rohith-JN.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Rohith-JN",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/RohithNambiar4",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on LinkedIn",
+        "url": "https://www.linkedin.com/in/rohith-nambiar-496a52240/",
+        "icon": "Linkedin"
+      },
+      {
+        "name": "Personal Porftolio",
+        "url": "https://rohithnambiar.in/",
+        "icon": "Linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started software development",
+        "date": "December 2019",
+        "icon": "globe",
+        "color": "blue",
+        "description": "Started software development (Python, Javascript, Flutter)",
```

I recommend capitalizing the `s` here. What do you think? 🙂
```suggestion
        "description": "Started software development (Python, JavaScript, Flutter)",
```